OIPA/iati/parser/IATI_2_03.py

```diff
@@ -3025,6 +3025,28 @@ def iati_activities__iati_activity__legacy_data(self, element):
 
         return element
 
+    # tag: condtions"""
+    def iati_activities__iati_activity__conditions(self, element):
+        conditions_attached = element.attrib.get('attached')
+
+        if not conditions_attached:
+            raise RequiredFieldError(
+                "conditions",
+                "attached",
+                "required attribute missing"
+            )
+        activity = self.get_model('Activity')
+        conditions = models.Conditions()
+        conditions.activity = activity
+        conditions.attached = self.makeBool(conditions_attached)
+
+        # It is impossible to assign related object (ForeignKey) before it's
+        # saved (later in other parser methods), so:
+        # XXX: not sure how efficient this is.
```

Comment:
I (out of curiosity) checked it without the `save()` method and it works without it, so this actually should be removed. Actually, we should try to avoid such `save()` method calls as much as possible, because then it hits the db every time, and with hundreds of Publishers and their XML files and `save()` calls it's gonna be a diseaster. This is actually (in theory) very inefficient :( See: #649

Reply:
(the test also passes without `save()` call)